<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Mithuns-Mac.local" failures="0" tests="3" name="tests.LoginTest" time="13.353" errors="1" timestamp="2024-05-03T22:38:33 IST" skipped="0">
  <testcase classname="tests.LoginTest" name="verifyLoginWithCorrectCredentials1" time="4.258"/>
  <system-out/>
  <testcase classname="tests.LoginTest" name="verifyLoginWithCorrectCredentials2" time="4.686"/>
  <system-out/>
  <testcase classname="tests.LoginTest" name="verifyLoginWithCorrectCredentials" time="4.409">
    <error type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException
at tests.LoginTest.verifyLoginWithCorrectCredentials(LoginTest.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
]]>
    </error>
  </testcase> <!-- verifyLoginWithCorrectCredentials -->
  <system-out/>
</testsuite> <!-- tests.LoginTest -->
